app/src/main/java/se/nullable/flickboard/ui/Keyboard.kt

```diff
@@ -168,7 +168,7 @@ fun Keyboard(
             }
         }
     }
-    val pointerTrailColor = MaterialTheme.colorScheme.onSurface
+    val pointerTrailColor = pointerTrailColour.value!!
```

Comment:
Fwiw, this will crash if no colour has been configured yet (`!!` is generally a pretty red flag, if it _is_ safe then the types should account for that instead, if possible).